prereise/gather/griddata/hifld/data_process/helpers.py

```diff
@@ -0,0 +1,23 @@
+from prereise.gather.griddata.hifld import const
+
+
+def map_state_and_county_to_interconnect(state_abv, county):
+    """Map a state and a county to an assumed interconnection.
+
+    :param str state_raw: two-letter state abbreviation.
+    :param str county: county name.
+    :raises ValueError: if the provided state abbreviation isn't present in mappings.
+    :return: (*str*) -- interconnection name.
+    """
+    # Coerce state abbreviation to upper-case
+    state_upper = state_abv.upper()
+    for region in ("east", "west"):
+        if state_upper in const.interconnect2state[region]:
+            return region
+    if state_upper in const.interconnect2state["split"]:
+        for region in set(const.state_county_splits[state_upper].keys()) - {"default"}:
+            if county.upper() in const.state_county_splits[state_upper][region]:
+                return region
+        return const.state_county_splits[state_upper]["default"]
+    else:
```

Comment:
I think we could drop the `else`

Reply:
Fair enough, this was more of a debug/development `else` so that I didn't get mysterious NoneType returns downstream. Do you think it's worth refactoring as an input check, or ditch completely?

Reply:
I was thinking just keep the error at the end, but remove the `else`, since it's implied. Makes sense to raise instead of letting it return `None`.

Reply:
Aha

Reply:
Done.